src/components/ToolbarDropdown/ToolbarDropdown.tsx

```diff
@@ -26,18 +27,18 @@ export class ToolbarDropdown extends React.Component<ToolbarDropdownProps, Toolb
   }
 
   onKeyChanged = (key: any) => {
-    this.setState({ currentValue: key[0], currentName: key[1] });
-    this.props.onClick(key[0]);
+    this.setState({ currentValue: key, currentName: this.props.options[key] });
+    this.props.onClick(key);
   };
 
   render() {
     return (
       <div className="form-group">
         {this.props.nameDropdown && <label style={{ paddingRight: '0.5em' }}>{this.props.nameDropdown}:</label>}
-        <DropdownButton title={this.state.currentName} onSelect={this.onKeyChanged} id="kiali-dropdown">
-          {this.props.options.map(r => (
-            <MenuItem key={r[0]} active={r[0] === this.state.currentValue} eventKey={r}>
-              {r[1]}
+        <DropdownButton title={this.state.currentName} onSelect={this.onKeyChanged} {...this.props}>
```

Comment:
I see here this is not changed, compared with the previous one.
It seems ok to me but just leaving a note as perhaps others may wonder the same.

Reply:
I added this.props to the component to propage to child....for example "id", with this you don't need to remove the tests 

https://github.com/kiali/kiali-ui/blob/master/src/components/MetricsOptions/__tests__/MetricsOptionsBar.test.tsx#L24